### ISSUE TITLE
Disco Pane content update - Dec. 2016 

### ISSUE THREAD

**Issue by devaneymoz** [mozilla/addons-server#4097]
For the **15 December** push, can we please REMOVE:

- Video DownloadHelper
- All three themes (Aurora Australis, Snow Style, My Vynil)

... and REPLACE them with YouTube High Definition and three new themes, respectively (note full copy included):

**Enhance YouTube** 
with [YouTube High Definition by Baris Derin ](https://addons.mozilla.org/firefox/addon/youtube-high-definition/)
Automatically play YouTube videos in high-def, turn off annotations, adjust player size, and many other ways to personalize your video-watching experience.  

[themes]

Ice Flower by [Sinine](https://addons.mozilla.org/firefox/addon/ice-flower-265447/)

Les Éléphants by [Charlotteo o](https://addons.mozilla.org/firefox/addon/les-%C3%A9l%C3%A9phants/)

Owl First Snow by [Candelora](https://addons.mozilla.org/firefox/addon/owl-first-snow/)

**Comment by ValentinaPC** [comment]
While setting about:addons to -dev, I saw some that the "Video DownloadHelper" and all three themes (Aurora Australis, Snow Style, My Vynil) were removed.
In replace, "Enhance YouTube  - with YouTube High Definition by Baris Derin " and "Ice Flower by Sinine" were added.
The other two themes are not available - I guess - because they are not listed in -dev, yes?
Postfix screenshot:
![2016-12-02_1041](https://cloud.githubusercontent.com/assets/15685960/20827633/e5a1f214-b87b-11e6-94fc-f2790a0e576e.png)

Just to be sure, we will recheck this on -prod after the 15 Dec. push.
